smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftSettings.kt

```diff
@@ -98,6 +98,7 @@ class SwiftSettings(
                     COPYRIGHT_NOTICE,
                     VISIBILITY,
                     FOR_PROTOCOL_TESTS,
+                    INTERNAL_CLIENT,
```

Comment:
Forgot to add this when I added the `internalClient` property... it silences a bunch of Kotlin codegen warnings.